src/test/a11y/a11y.ts

```diff
@@ -23,23 +23,31 @@ app.locals.csrf = 'dummy-token'
 const cookieName: string = config.get<string>('session.cookieName')
 
 const agent = supertest.agent(app)
-const pa11yTest = pa11y({
-  page: {
+
+interface Issue {
+  type
+}
+
+async function runPa11y (url: string): Promise<Issue[]> {
+  const result = await pa11y(url, {
     headers: {
       Cookie: `${cookieName}=ABC`
+    },
+    chromeLaunchConfig: {
+      args: ['--no-sandbox']
```

Comment:
Needed to run puppeteer on Travis without any additional changes to the build process.

See: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-on-travis-ci